packages/lit-helpers/package.json

```diff
@@ -36,5 +37,9 @@
   "peerDependencies": {
     "lit": "^2.0.0"
```

Comment:
Do we actually need to vend multiple exports/add multiple hard dependencies, or can we just loosen this `peerDependency` to include `^2.0.0 || ^3.0.0` so that the consuming project can continue to outline the versions of the dependencies they need rather than forcing both versions that the change does now?

Reply:
**sigh** I switched laptops and forgot to put the only change that was actually needed... new commit updates the "peerDependencies", and I updated "dependencies" to "devDependencies"